pandas/_libs/period.pyx

```diff
@@ -19,8 +18,11 @@ from pandas import compat
 from pandas.compat import PY2
 
 cimport cython
+# this is _libs.src.datetime, not python stdlib
```

Comment:
Similar comments exist elsewhere e.g. in lib.pyx:

```
# this is our tseries.pxd
from datetime cimport *
```

This name overlap is an easy source of confusion.

Not worth arguing about so I'll change it.  Just sayin'.